protos/sigstore_trustroot.proto

```diff
@@ -87,6 +93,12 @@ message CertificateAuthority {
         // The TimeRange should be considered valid *inclusive* of the
         // endpoints.
         dev.sigstore.common.v1.TimeRange valid_for = 4;
+        // The name of the operator of this certificate authority. This MUST
+        // be used when there are multiple timestamp authorities to determine
+        // if the signed timestamp verification meets a specified threshold,
+        // e.g. two signed timestamps from timestamp authorities operated by
+        // the same operator should count as only one valid timestamp.
+        string operator = 5;
```

Comment:
I think this part is maybe a bit confusing, because we re-use CertificateAuthority for TSAs.

Should we switch TSA to it's own type?

Reply:
I'm not sure it would be more clear if the TSA was it's own type? It would be a pretty big change for the clients if we did. I'm inclined to keep as is.

Reply:
Agreed, if we were to revisit this and cut a new version, then I'd be more inclined, but for now, I'd keep it as is. I'll update the comment a bit to acknowledge this. Also, having an operator for a CA would allow a client to potentially filter out operators it doesn't want to use.